### PR TITLE
#156958154 customer onboarding tour

### PR DESCRIPTION
**Pull request by topseySuave**
### What does this PR do?
Guides a new user through a shop

### Description of Task to be completed?
When a new user navigate to a shop he/she is automatically inducted
An existing user can click the `Take a tour` button.

### How should this be manually tested?
npm run 
```
git clone https://github.com/andela/nazgul-rc.git
``` 
to clone the repository
navigate to the ```nazgul-rc``` folder
run
```
 curl https://install.meteor.com/ | sh
```
then run 
```
npm install -g reaction-cli
```
install dependencies with 
```
meteor npm install
```
run  ```reaction``` to start up the server
once the app is running, navigate to the app using ```localhost:3000```
A button at the top navbar is visible saying `Take a tour`
Clicking this button will start the tour.

### What are the relevant pivotal tracker stories?
link to relevant PT story: https://www.pivotaltracker.com/story/show/156958154

### CODE REVIEW

File: imports/plugins/included/product-variant/components/productGrid.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 import startTour from "../../adminTour";
+import onboarding from "../../onboarding/onboarding";
 
 class ProductGrid extends Component {
   static propTypes = {
     products: PropTypes.array
   };
 
   componentDidMount() {
+    if (!Reaction.hasPermission("admin")) {
```
Comment (morekoya) [inline imports/plugins/included/product-variant/components/productGrid.js:14]: Does this mean that only admins can take the tour?

Comment (topseySuave) [reply imports/plugins/included/product-variant/components/productGrid.js:14]: No it does not..

**Comment by morekoya** [comment]
**App Review:**
1) There's a typo on this card
screen shot 2018-05-24 at 11 42 36 am

2) Empty description
screen shot 2018-05-24 at 11 44 30 am

3) There's no `Start Onboarding` to click on as directed
screen shot 2018-05-24 at 11 45 05 am

4) Is there any particular reason why the cards do not follow the nav menu items in order?

**Comment by topseySuave** [comment]
Will correct all that now...

**Comment by topseySuave** [comment]
### Corrections :
image

2:
image

3:
image

**Comment by morekoya** [comment]
The first and second points are still not grammatically correct.

**Comment by topseySuave** [comment]
Been worked on

 ### correct third issue:
image

**Comment by topseySuave** [comment]
### 1:
image


### 2: 
image

**Comment by morekoya** [comment]
It's supposed to be a full-stop. `This is the end of the tour.`

**Comment by topseySuave** [comment]
Am sorry corrected

image

**Pull request merged**